documentation/modules/ROOT/pages/connectors/oracle.adoc

```diff
@@ -1100,23 +1100,30 @@ If _truncate_ events are not desired, they can be filtered out with the xref:ora
 [[oracle-data-type-mappings]]
 == Data type mappings
 
-To represent changes that occur in a table rows, the {prodname} Oracle connector emits change events that are structured like the table in which the rows exists.
-The event contains a field for each column value.
-Column values are represented according to the Oracle data type of the column.
-The following sections describe how the connector maps oracle data types to a _literal type_ and a _semantic type_ in event fields.
+When the {prodname} {connector-name} connector detects a change in the value of a table row, it emits a change event that represents the change.
+Each change event record is structured in the same way as the original table, with the event record containing a field for each column value.
+The data type of a table column determines how the connector represents the column's values in change event fields, as shown in the tables in the following sections.
 
-_literal type_:: Describes how the value is literally represented using Kafka Connect schema types: `INT8`, `INT16`, `INT32`, `INT64`, `FLOAT32`, `FLOAT64`, `BOOLEAN`, `STRING`, `BYTES`, `ARRAY`, `MAP`, and `STRUCT`.
+For each column in a table, {prodname} maps the source data type to a _literal type_ and, and in some cases, a _semantic type_, in the corresponding event field.
 
-_semantic type_:: Describes how the Kafka Connect schema captures the _meaning_ of the field using the name of the Kafka Connect schema for the field.
+Literal types:: Describe how the value is literally represented, using one of the following Kafka Connect schema types: `INT8`, `INT16`, `INT32`, `INT64`, `FLOAT32`, `FLOAT64`, `BOOLEAN`, `STRING`, `BYTES`, `ARRAY`, `MAP`, and `STRUCT`.
+
+Semantic types:: Describe how the Kafka Connect schema captures the _meaning_ of the field, by using the name of the Kafka Connect schema for the field.
+
+For some Oracle binary large object (BLOB) and numeric data types, you can manipulate the way that the connector performs the type mapping by changing default configuration property settings.
```

Comment:
```suggestion
For some Oracle large object (CLOB, NCLOB, and BLOB) and numeric data types, you can manipulate the way that the connector performs the type mapping by changing default configuration property settings.
```
It's important to note that large object data types can be either binary or character-based and the character based can either be single-byte (CLOB) or multi-byte (NCLOB) formats.  So feel free to adjust how we convey that in the above sentence but I don't think the usage of "Oracle binary large object" conveys all the right bits as it implies exclusion to the character-based flavors.